test/fixtures/es-module-loaders/js-loader.mjs

```diff
@@ -1,10 +1,11 @@
-import _url from 'url';
+import { URL } from 'url';
```

Comment:
Side note: This is global now, so we could just remove this line and/or split this change into a separate PR.

Reply:
i left it in just in case these changes get backported to other release lines